src/components/organisms/Settings/ProjectList/index.tsx

```diff
@@ -29,17 +33,73 @@ const ProjectList: React.FC<Props> = ({ teamId }) => {
     assetModalOpened,
     toggleAssetModal,
     onAssetSelect,
+    getMoreProjects,
   } = useHooks(teamId);
 
+  const projectLength = useMemo(
+    () => ({
+      currentProjectsLength: currentProjects.length || 0,
+      archivedProjectsLength: archivedProjects.length || 0,
```

Comment:
Unnecessary because currentProjects and archivedProjects are arrays and never undefined
```suggestion
      currentProjectsLength: currentProjects.length,
      archivedProjectsLength: archivedProjects.length,
```